custom/icds_reports/ucr/data_sources/child_health_cases.json

```diff
@@ -1283,6 +1283,53 @@
         "column_id": "is_availing",
         "is_nullable": true,
         "datatype": "integer"
+      },
+      {
+        "column_id": "aadhar_date",
```

Comment:
This is:

```python
if person_case.aadhar_number != None:
   if person_case.aadhar_date != None:
       return person_case.aadhar_date
   else:
       return (expensive query to get the date from forms)
else:
   return None
```

Very similar to https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/ucr/data_sources/child_health_cases_monthly_tableau2.json#L575